### ISSUE TITLE
Strict no null {{ expression }}

### ISSUE THREAD

**Issue by dukefong8** [unifire-app/aspect#9]
**Is your feature request related to a problem? Please describe.**
I want aspect throw an error like "{{ Foo.Bar }} is null in file.tmpl:line:col" when {{ Foo.Bar }} evaluates to null.

**Describe the solution you'd like**
I'd like to try it out with my local clone first. However, I'm not sure where the relevant code are. Can you please give some guidance, e.g. which function, or lines etc.

**Describe alternatives you've considered**
maybe control it via an option 'strictNil'

**Additional context**
Basically, make Aspect behave like Go template, 

```
template: view.tmpl:10:23: executing "main" at <.Bar>: can't evaluate field Bar in type Foo
```
